run-build-functions.sh

```diff
@@ -135,7 +135,7 @@ run_yarn() {
   yarn_version=$1
   restore_home_cache ".yarn_cache" "yarn cache"
 
-  if ! [ $(which corepack) ]; then
```

Comment:
I think you'll need to pass the `featureFlags` argument:

E.g.
- https://github.com/netlify/build-image/blob/27a7202581435348546b4aa1a3a63e1ed1da38c2/run-build-functions.sh#L221-L223
- https://github.com/netlify/build-image/blob/27a7202581435348546b4aa1a3a63e1ed1da38c2/run-build-functions.sh#L611